src/main/java/com/example/overworldbackend/controller/OverworldController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.overworldbackend.controller;
+
+import com.example.overworldbackend.baseClasses.Configuration;
+import com.example.overworldbackend.repositories.OverworldRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Optional;
+
+
+@RestController
+public class OverworldController {
+
+    @Autowired
+    OverworldRepository overworldRepository;
+
+    @PostMapping("/save-minigame-config")
```

Comment:
please use better mappings. Maybe something like
```suggestion
    @PostMapping("/minigame/config")
```